src/components/pages/about-us/impressions.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  AboutUsImpressionTileSize,
+  ContentfulAboutUsImpression,
+} from '../../../types';
+import { SectionHeader } from '../../content/section-header/section-header';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import styled from 'styled-components';
+import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+import { up } from '../../support/breakpoint';
+
+interface ImpressionsProps {
+  impressions: ContentfulAboutUsImpression[];
+}
+
+const GalleryGrid = styled.div`
+  display: grid;
+  gap: 8px;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(auto-fill, 250px);
+  grid-auto-flow: row;
+
+  margin-top: 48px;
+
+  ${up('sm')} {
+    gap: 24px;
+    margin-top: 60px;
+  }
+`;
+
+const getSpan = (tileSize: AboutUsImpressionTileSize) => {
+  switch (tileSize) {
+    case 'landscape-big':
+      return 'span 2 / span 2';
+    case 'landscape-small':
+      return 'span 1 / span 1';
+    case 'portrait':
+      return 'span 2 / span 1';
+  }
+};
+
+const GalleryItem = styled.div<{ tileSize: AboutUsImpressionTileSize }>`
+  grid-area: ${({ tileSize }) => getSpan(tileSize)};
+`;
+
+const GalleryImage = styled(GatsbyImage)`
+  min-width: 100%;
+  min-height: 100%;
+`;
+
+export const Impressions = ({ impressions }: ImpressionsProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <SectionHeader
+        headline={t<string>('about-us.impressions.heading')}
+        kicker={t<string>('about-us.impressions.title')}
+      >
```

Comment:
That's missing:
![Screenshot 2023-03-03 at 12 28 33](https://user-images.githubusercontent.com/1701755/222710215-102b1900-2799-4746-bdb8-c426b0bbab8b.png)


In the PR you said that content follows. But I think it's not acceptable to merge content that does not hold any content and instead the translation keys. Can you share your approach in detail?

Reply:
I'm sorry I did not recognize that you merge into a different branch. That's 100% fine then. I though I saw the main branch there when I wrote this comment.